aten/src/ATen/cuda/CUDAGraph.h

```diff
@@ -85,8 +91,11 @@ struct TORCH_CUDA_CPP_API CUDAGraph {
   // RNG state trackers
   at::Tensor seed_extragraph_;
   at::Tensor offset_extragraph_;
+  std::vector<void*> const_buffers_;
```

Comment:
cc @eellison is that an ok thing to do? I know you worked pretty hard to remove all allocations here including workspaces

Reply:
Is this something that we want to have generically in CUDA graphs or would it make more sense to have it as something that `MultiTensorApply` holds on to if it is to be used in a graph capture?
I guess I'm missing some context on why this allocation has to be invoked by the graph itself.

Reply:
> I know you worked pretty hard to remove all allocations here including workspaces

The particular problem I had was for allocations which were made with the cuda caching allocator and which would persist outside of the scope of reinvoking the cuda graph. E.g., during capture adding a tensor to cublas workspace which would could then get potentially used in egaer.

 So that doesn't apply here right ?

Reply:
> Is this something that we want to have generically in CUDA graphs

Hey @eqy, this is not a goal (it might even be something we want to avoid). After seeing @eellison's comment, I think I may have been overcomplicating things. The goal is to allocate a buffer and tie its lifetime to the capturing `CUDAGraph` object, and it's probably achievable by simply allocating it with `CUDACachingAllocator::raw_alloc` and letting it "leak".